src/main/java/io/temporal/samples/hello/HelloActivity.java

```diff
@@ -33,34 +33,64 @@
 import java.time.Duration;
 
 /**
- * Hello World Temporal workflow that executes a single activity. Requires a local instance the
- * Temporal service to be running.
+ * Sample Temporal workflow that executes a single workflow activity.
+ *
+ * <p>To execute this example a locally running Temporal service instance is required. You can
+ * follow instructions on how to set up your Temporal service here:
+ * https://github.com/temporalio/temporal/blob/master/README.md#download-and-start-temporal-server-locally
  */
 public class HelloActivity {
 
-  static final String TASK_QUEUE = "HelloActivity";
+  // Define the task queue name
+  static final String TASK_QUEUE = "HelloActivityTaskQueue";
+
+  // Define our workflow unique id
+  static final String WORKFLOW_ID = "HelloActivityWorkflow";
```

Comment:
Should we mention about workflow id reuse policy here?

Reply:
i added a workflow id to each example so that in the web ui it does not display the unique generated one. when running multiple examples locally it helps with figuring out which workflow run is associated with what example.
can you show me maybe what this workflow reuse policy is so i can add ? you mean on like on continue run or something else ?

Reply:
See https://docs.temporal.io/docs/concept-workflows/#workflow-id